### PR TITLE
fix(content-explorer): Request fields for Preview SDK file cache

### PR DESCRIPTION
**Pull request by sahiga**
This PR updates the folder request fields in the Content Explorer to include additional fields required by the Preview SDK cache:
- file_version
- sha1
- shared_link
- watermark_info

### CODE REVIEW

**Review by jstoffan — APPROVED** [review]
Is there any performance impact on the associated calls after this change? It looks like this code is used by ContentPicker, as well, which doesn't support preview.

**Comment by sahiga** [comment]
Good call, @jstoffan. I created a separate array for the Content Explorer folder fields.

File: src/utils/fields.js [hunk 1]
```diff
@@ -80,9 +79,15 @@ const FOLDER_FIELDS_TO_FETCH = [
     FIELD_AUTHENTICATED_DOWNLOAD_URL,
     FIELD_IS_DOWNLOAD_AVAILABLE,
     FIELD_REPRESENTATIONS,
+    FIELD_URL,
+];
+
+// Fields needed for Content Explorer folder requests
+const CONTENT_EXPLORER_FOLDER_FIELDS_TO_FETCH = [
```
Comment (jstoffan) [inline src/utils/fields.js:86]: Should this constant live in a file within the `src/elements/content-explorer` directory?

Comment (sahiga) [reply src/utils/fields.js]: Yes! Just moved the constant into the `src/elements/content-explorer` directory.

**Review by jstoffan — APPROVED** [review]
Thanks for fixing this issue!

**Review by swfree — APPROVED** [review]

**Pull request merged**